### ISSUE TITLE
How to do multiline string with curly braces

### ISSUE THREAD

**Issue by dwach414** [jonsamwell/flutter_gherkin#150]
When Trying to create a step using a multiline string if there are curly braces it will fail the pub build. It seems to work if you escape the first curly brace but only when it's on line. This makes it really hard when dealing with large json strings.

For example:
This works:
```
Given I have item with data
    """
    \{"data": "testData"}
    """
```

But this won't work:
```
Given I have item with data
    """
    \{
    "data": "testData"
    }
    """
```

It's really annoying to have to put a huge json object on one line. Any help would be appreciated.

**Comment by jonsamwell** [comment]
What is the build error message?

**Comment by dwach414** [comment]
The error that we get back has a lot of random stuff in it. Looks kinda of like this. Using 
```
    """
    \{
    "canSubmit":  "true"
    }
    """
```

```
[SEVERE] flutter_gherkin:gherkin_test_suite on integration_test/gherkin_suite_test.dart:
An error `FormatterException` occurred while formatting the generated source for
  `asset:casedoc/integration_test/gherkin_suite_test.dart`
which was output to
  `integration_test/gherkin_suite_test.gherkin_tests.g.part`.
This may indicate an issue in the generator, the input source code, or in the
source formatter.
Could not format because the source could not be parsed:

line 116, column 5 of .: Expected a class member.
    ╷
116 │     );
    │     ^
    ╵
line 83, column 14 of .: Expected to find ')'.
   ╷
83 │   "canSubmit": "true"
   │              ^
   ╵
line 116, column 6 of .: Expected a class member.
    ╷
116 │     );
    │      ^
    ╵
line 39, column 1 of .: Expected to find ']'.
   ╷
39 │ }'],
   │ ^
   ╵
line 39, column 2 of .: Expected to find ','.
   ╷
39 │ }'],
   │  ^^^
   ╵
line 115, column 8 of .: Expected a class member.
    ╷
115 │       },
    │        ^
    ╵
line 87, column 3 of .: Expected to find ']'.
   ╷
87 │   );
   │   ^
   ╵
line 37, column 16 of .: Unterminated string literal.
   ╷
37 │     <String>['\{
   │                ^
   ╵
line 39, column 4 of .: Unterminated string literal.
   ╷
39 │ }'],
   │    ^
   ╵
line 38, column 14 of .: Expected an identifier.
   ╷
38 │   "canSubmit": "true"
   │              ^
   ╵
(9 more errors...)
[INFO] 15.0s elapsed, 255/270 actions completed.
[INFO] 16.1s elapsed, 306/321 actions completed.
[INFO] Running build completed, took 16.8s

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 93ms

[SEVERE] Failed after 16.9s
```

**Comment by jonsamwell** [comment]
Fixed in 3.0.0-rc.3
